src/main/res/layout/activity_server_file_video.xml

```diff
@@ -31,17 +31,28 @@
         android:layout_height="wrap_content"
         android:layout_gravity="center" />
 
+
     <FrameLayout
         android:id="@+id/layout_content"
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
+
         <SurfaceView
             android:id="@+id/surface"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
             android:layout_gravity="center"
             android:fitsSystemWindows="false" />
+        <Button
+            android:id="@+id/button"
+            android:layout_width="20dp"
+            android:layout_height="20dp"
+            android:background="@drawable/rotation"
+            android:layout_marginTop="80dp"
```

Comment:
This top margin of 80dp looks odd. According to me, it should be something like this: the top margin can be 20dp initially and the button should slide up and down as the controls are shown or hidden. This may be done by some use case of `android:fitsSystemWindows="true"`. Please check if you can make it work like this.